mautic/mautic
# [UX] Create a tooltip for "Custom CSS" to make sure the user does not include <style> HTML tags

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Perhaps this modification is a little more on the side of excessive security but it can be positive to avoid problems with items being deconfigured when put live.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Focus items
3. Create a new 
4. Open the Builder > Content
5. Look at the Custom CSS label, it has a tooltip

![image](https://github.com/mautic/mautic/assets/116097999/b618dbef-d0dc-4388-8599-168e18332ce5)
